### PR TITLE
fix: Missing choice in csv

### PR DESCRIPTION
**Pull request by ChaituVR**
If we download csv from https://snapshot.org/#/friendswithbenefits.eth/proposal/0xafe3a0426d4e6c645e869707f1b581765698d80c8d3e9cd37d7d3bf5e6f894e7 you will see missed choice because we are adding value to `-1` index

### CODE REVIEW

File: src/lib/votesReport.ts [hunk 1]
```diff
@@ -100,8 +100,14 @@ class VotesReport extends Cache {
 
     if (typeof vote.choice !== 'number' && this.proposal) {
       choices = Array.from({ length: this.proposal.choices.length });
-      for (const [key, value] of Object.entries(vote.choice)) {
-        choices[parseInt(key) - 1] = value;
+      if (Array.isArray(vote.choice)) {
+        vote.choice.forEach((value, index) => {
+          choices[index] = value;
+        });
```
Comment (wa0x6e) [inline src/lib/votesReport.ts:106]:
```suggestion
        choices = vote.choice;
```

**Review by wa0x6e — CHANGES_REQUESTED** [review]

File: src/lib/votesReport.ts (hunk 1 above)
Comment (ChaituVR) [reply src/lib/votesReport.ts:106]: For approval voting, there is a possibility that user can pick less number of choices 
```js
[3, 4]
```

Comment (ChaituVR) [reply src/lib/votesReport.ts:106]: In this case, the other choices should be empty if we use `choices = vote.choice;` it will show wrong output on csv

Comment (ChaituVR) [reply src/lib/votesReport.ts:106]: Better I will add tests for approval and quadratic 😄

**Review by wa0x6e — APPROVED** [review]
utAck

**Pull request merged**